### PR TITLE
Emergency release 3.0.1

### PR DESCRIPTION
**Pull request by Anduin2017**

### CODE REVIEW

**Review by xxyzz — APPROVED** [review]

**Comment by Anduin2017** [comment]
@xxyzz  How about merging it after fixing #285?

Maybe I got an idea: Only allows using OAuth system to sign in for `PWA`, and only allows password authentication for `Electron` and `Cordova`. How do you like that?

File: src/app/Styles/menu.css [hunk 1]
```diff
@@ -82,7 +82,7 @@
 }
 
 :host-context(.theme-dark) .menu-button:hover {
-    background-color: #576574 ;
+    background-color: #374554 ;
```
Comment (l2dy) [inline src/app/Styles/menu.css]: Nitpick: remove the space before `;`.

Comment (Anduin2017) [reply src/app/Styles/menu.css]: Will fix. Thanks!

**Review by xxyzz — APPROVED** [review]
...

**Review by hv0905 — APPROVED** [review]

**Pull request merged**